docs/guides/admin/docs/migration-domain-https.md

```diff
@@ -0,0 +1,44 @@
+# Domain change/HTTPS migration
+
+If you move your Opencast server to a new domain or if you switch from HTTP to HTTPS, you will change
+`org.opencastproject.server.url` in `etc/custom.properties`.
+But if your Opencast already processed events, these events still contain the old URL in various places.
+So in order for old events to work correctly, you have to migrate them.
+Re-processing or re-publishing will update the links but this may not be an option if you have many events.
+In that case, the following steps might help.
+
+> Note that you modify stored data directly without any safety nets usually provided by Opencast. You should understand
+> what you are doing!
+
+1. Backup your database, and the Solr and OpenSearch/Elasticsearch indexes!
+    - OpenSearch/Elasticsearch: `{data}/index`
+    - Solr: `{data}/solr-indexes`
+2. Change `org.opencastproject.server.url` and test your set-up with a new publication (i.e. uploading a video).
+3. Put all your nodes into maintenance mode or, at least, do not process any videos.
+4. Update the media packages on disk.
+   Run the following in the folder configured in `org.opencastproject.storage.dir` (usually your NFS).
+   Note: only the subfolders `archive/` and `downloads/` will be affected by this, so you can also run it in those two
+   individually.
+   This is especially useful to reduce the execution time of the command if your NFS contains other, Opencast-unrelated
+   files (e.g. `.snapshots/`).
+
+         # This changes the doman AND from http to https. Carefully adjust the command as needed!
+         find . -type f -name "*.xml" -exec \
+            sed -i 's/http\:\/\/old-domain\.example\.com\:80/https:\/\/new-domain.example.com/g' {} +
```

Comment:
I would be tempted to add a bunch more explanation here in terms of what the old-domain value *could* look like.  Your example here has the port explicitly listed, but I'd bet a bunch won't.  But some will, and if you don't handle both case we'll have support cases showing up wondering what happened.  Perhaps we grep for the old value, if there's a way, and then have them use that?

Also, `sed -E` to remove the escaping would make end-user life easier IMO.

Reply:
Mhh I'm not sure about this. I don't have the time to develop a script to automatically extract the current domain name. I think with the comment above the command and the previous text mentioning `org.opencastproject.server.url`, it should be fairly clear? If you have any specific ideas how to improve this, let me know though.

The `sed -E` thing I changed... I hope this is what you meant.